src/style/mixins.scss

```diff
@@ -340,3 +340,24 @@
         }
     }
 }
+
+/**
+* This mixin creates a cross-browser font stack.
+* - `sans-serif` can be used for the UI of the components.
+* - `monospace` can be used for code.
+*
+* ⚠️ If we change the font stacks, we need to update
+* 1. the consumer documentation in `README.md`, and
+* 2. the CSS variables of `--kompendium-example-font-family`
+* in the `<style>` tag of `index.html`.
```

Comment:
I added some fixups that places the addition of parts of this comment in the commits where the things referred to in the comment are introduced. For future traceability.